11ty/definitions/nl_NL/biseksueel.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Biseksueel
+slug: biseksueel
+speech: adj
+defined: true
+---
+
+het zijn, relateren aan of zich laten kenmerken door seksueel aangetrokken zijn tot meer dan een geslacht.
+
+#### Opmerking
+
+Biseksualiteit sluit aangetrokken zijn tot [non-binaire](#non-binair) personen of [transgenders](#transgender) niet uit.
```

Comment:
"Transgenders" doesn't feel right for the plural form. By separating it from "personen" it also implies that "transgender" isn't part of "personen". Perhaps rewrite to "Biseksualiteit sluit aangetrokken zijn tot non-binaire of transgender personen niet uit"